src/app/view/components/dot-persona-selector-option/dot-persona-selector-option.component.html

```diff
@@ -0,0 +1,13 @@
+<dot-avatar
+    [label]="persona.name"
+    [showDot]="persona.personalized"
+    [size]="24"
+    [url]="persona.photo"
+></dot-avatar>
+<label [ngClass]="{ 'dot-persona-selector-option__personalized': persona.personalized }">{{
+    persona.name
+}}</label>
+<button *ngIf="persona.personalized" (click)="deletePersonalized($event)">
+    {{ dotMessageService.get('modes.persona.personalized') }}
```

Comment:
Create a prop like: https://github.com/dotCMS/core-web/blob/master/src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts#L348 is not good to call methods from the html template, is old code where we do it.